test/docker-compose.yml

```diff
@@ -3,14 +3,16 @@ services:
   consul:
     image: "consul:0.8.3"
     hostname: "consul"
-    privileged: true
```

Comment:
Maybe we need to run etcd with `privileged: true`? :/

Reply:
It works fine in my local ginkgo vm

Reply:
Docker-Compose is ran on the host on Jenkins, not in the Ginkgo VM. Are you running the unit tests within the Ginkgo VM itself?